src/main/resources/assets/corpse/lang/pl_pl.json

```diff
@@ -17,5 +17,6 @@
   "tooltip.corpse.death_date": "Gracz zmarł %s",
   "tooltip.corpse.item_count": "Zawiera stacków: %s",
   "button.corpse.transfer_items": "Przenieś przedm.",
-  "button.corpse.additional_items": "Dodatkowe rzeczy"
+  "button.corpse.additional_items": "Dodatkowe rzeczy",
+  "config.jade.plugin_corpse.corpse": "Zwłoki"
```

Comment:
I assumed that "Corpse" ("Zwłoki" in Polish) in this case is just the name of the entity, not a proper noun (i.e. the name of the mod) and I translated it under that assumption. Is it correct?

Reply:
That should be fine. I don't even know why Jade needs this string...